### ISSUE TITLE
Lineups are only populating for first and current week matchups 

### ISSUE THREAD

**Issue by wsecheng** [cwendt94/espn-api#524]
### Sport

Basketball

### Summary

I'm trying to pull the players + stats that participated in every matchup throughout the year, but I'm seeing that `home_lineup` and `away_lineup` attributes only return a non-empty list for teams in the first and current week matchups.

### Logs/Data

```
for week in range(league.currentMatchupPeriod):
    for matchup in league.box_scores(matchup_period = week+1):
        print(week)
        print(matchup)
        for player in matchup.home_lineup:
            print(player)

        for player in matchup.away_lineup:
            print(player)
```

**Comment by cwendt94** [comment]
what type of scoring league is this? I am able to see all the lineups for my H2H points league.

**Comment by wsecheng** [comment]
It's a H2H cats league

**Comment by cwendt94** [comment]
Okay, I tried by cats league and seeing the same issue. I will take a look.

**Comment by cwendt94** [comment]
So the issue is mapping the `matchup_period ` to a `scoring_period`. For CAT leagues it seems like the way we find scoring period does not work the same for H2H points. Right now I can't seem to find out how to see all the scoring periods in a matchup. To get around this issue for now if you also define a `scoring_period` to each of your matchup calls it should get the lineups correctly.

**Comment by wsecheng** [comment]
What `scoring_period` meant to represent and what should it be set to?

**Comment by cwendt94** [comment]
Yeah the `scoring_period` is the actual day of games being played. So the opening day of games for the season `scoring_period=1` second day of games `scoring_period=2`. And it keeps increasing for every day there are games.

H2H scoring had a way to get all of the scoring periods in a matchup but right now I can't find that data when the league is CAT
